src/app/core/models/templates/template.model.ts

```diff
@@ -30,6 +30,13 @@ export abstract class Template {
     this.regex.lastIndex = 0;
     return numOfMatch >= this.headers.length;
   }
+
+  parseErrorMessage(headers: Header[]): string {
+    const headerNames = headers.map((header) => " " + header.name);
+    const message = "Unable to find a Github comment with these headers:" + headerNames + '\n' +
+        "Please edit the comment manually on GitHub.";
+    return message;
+  }
```

Comment:
Currently, it seems that even when the template is missing only one header, all the header names get 
printed out. Can refer to [this issue](https://github.com/CATcher-org/pe-results/issues/299) for an example! 

Perhaps we can consider using some regex to find out which headers are missing.

Think we can discuss about this sometime this week!

Reply:
ah good point will look into it thanks!